editor/src/components/canvas/canvas-strategies/strategies/flow-reorder-strategy.spec.browser2.tsx

```diff
@@ -334,11 +458,38 @@ describe('Flow Reorder Strategy (Mixed Display Type)', () => {
     const strategies = renderResult.getEditorState().strategyState.sortedApplicableStrategies
 
     const flowReorderNotInStrategies = Array.isArray(strategies)
-      ? strategies.findIndex((strategy) => strategy.name === 'FLOW_REORDER')
+      ? strategies.findIndex((strategy) => strategy.strategy.id === 'FLOW_REORDER')
```

Comment:
Also here we were filtering the applicable strategies by comparing our expected ID against the name.